var/spack/repos/builtin/packages/protobuf/package.py

```diff
@@ -133,28 +118,25 @@ def cmake_args(self):
             args.extend(["-DCMAKE_MACOSX_RPATH=ON"])
         return args
 
+    @property
+    def root_cmakelists_dir(self):
+        if self.spec.satisfies("@:3.20"):
+            return join_path(self.stage.source_path, "cmake")
+        else:
+            return self.stage.source_path
+
     @when("@3.0.2:")
     def install(self, spec, prefix):
```

Comment:
You should be able to delete this whole `install` method and use the builtin one.